api/definitions.yaml

```diff
@@ -322,6 +322,8 @@ ExtendedProfile:
       $ref: "#/PreferredLanguages"
     is_inbox_enabled:
       $ref: "#/IsInboxEnabled"
+    tos_version:
```

Comment:
perhaps `accepted_tos_version` is better, to make it clear that the user has accepted it